### PR TITLE
Lesson 16 Gokhan

### PR DESCRIPTION
**Pull request by G-Memis**

### CODE REVIEW

File: Lesson16/Gokhan/script.js [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+    personAge(77);
+
+
+// 10. Write a function that takes age as a age and whether they are a student (yes or no). 
+//     If the age is less than 18 and the user is a student, print "You are a student and not old enough to vote". 
+//     If the age is between 18 and 65 (inclusive) and the user is not a student, print "You are eligible to vote". 
+//     If the age is greater than 65, print "You are a senior citizen". 
+//     If the user is a student but their age is greater than or equal to 18, print "You are old enough to vote".
+
+function checkVotingEligibility(age, isStudent) {
```
Comment (annasvst) [inline Lesson16/Gokhan/script.js:124]: For values like 'yes'/'no', in programming it's common to use Boolean type value. So change function body like this:

```
if (age < 18 && isStudent) {
        console.log("You are a student and not old enough to vote");
    } else if (age >= 18 && age <= 65 && !isStudent) {
        console.log("You are eligible to vote");
    } else if (age > 65) {
        console.log("You are a senior citizen");
    } else if (isStudent && age >= 18) {
        console.log("You are old enough to vote");
    }
```

See more info about booleans here: https://www.classes.cs.uchicago.edu/archive/2021/spring/11111-1/happycoding/p5js/if-statements.html#:~:text=An%20if%20statement%20checks%20a,when%20that%20value%20is%20true%20.

When calling the function, you need to call it like this: `checkVotingEligibility(17, true);`

**Review by annasvst — CHANGES_REQUESTED** [review]

File: Lesson16/Gokhan/script.js (hunk 1 above)
Comment (G-Memis) [reply Lesson16/Gokhan/script.js:124]: It is Done!

File: Lesson16/Gokhan/script.js [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+        console.log("You are eligible to vote");
+    } else if (age > 65) {
+        console.log("You are a senior citizen");
+    } else if (isStudent && age >= 18) {
+        console.log("You are old enough to vote");
+    }
+
+}
+
+checkVotingEligibility(16, "yes");
```
Comment (annasvst) [inline Lesson16/Gokhan/script.js]: When calling the function, you need to pass the value of the second parameter as boolean:
```suggestion
checkVotingEligibility(16, true);
```

File: Lesson16/Gokhan/script.js [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+checkVotingEligibility(40, "no");
```
Comment (annasvst) [inline Lesson16/Gokhan/script.js]:
```suggestion
checkVotingEligibility(40, false);
```

File: Lesson16/Gokhan/script.js [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+checkVotingEligibility(65, "no");
```
Comment (annasvst) [inline Lesson16/Gokhan/script.js]:
```suggestion
checkVotingEligibility(65, false);
```

File: Lesson16/Gokhan/script.js [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+checkVotingEligibility(22, "yes");
```
Comment (annasvst) [inline Lesson16/Gokhan/script.js]:
```suggestion
checkVotingEligibility(22, true);
```

**Review by annasvst — APPROVED** [review]
Approved

**Pull request merged**